src/models/Channel.ts

```diff
@@ -36,4 +37,8 @@ Channel.belongsTo(User, {
   as: 'owner',
 });
 
-export default Channel;
+export type ChannelModelStatic = typeof Model & {
```

Comment:
Looks good codes, just I didn't understand what does the `Static` mean? :?

Reply:
You can read the above `sequelize` link I've mentioned.
![image](https://user-images.githubusercontent.com/27461460/72342960-171d7680-3711-11ea-85b3-6c9319d11d15.png)

Reply:
Above also causes a problem when we are trying to pass `sequelize` definition through the graphql context.

Reply:
Oh, sorry for that I missed the links. Thanks! :D